### DISCUSSION TITLE
"GitHub Pages 404 Error for 'Project & Skills' Page Deployment from Navbar"

### DISCUSSION THREAD

**Discussion by ghost** [community/community#63999] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

I am encountering persistent issues with the deployment of my "Project & Skills" page on GitHub Pages. Despite following the proper setup and configuration steps, the page is not accessible and displays a "404 - Not Found" error when accessed through the GitHub Pages URL.

Details:

Repository URL: My GitHub Pages repository URL is: https://github.com/username/repository-name.
Branch for Deployment: I have configured GitHub Pages to deploy from the main (or appropriate) branch, where the HTML file and related assets for the "Project & Skills" page are located.
File Paths: I have verified that the file paths for HTML, CSS, images, and other assets are accurate and match the repository structure.
Case Sensitivity: I have ensured that file names and paths are correctly formatted and match the case sensitivity requirements of GitHub Pages.
GitHub Pages Settings: I have reviewed and confirmed the GitHub Pages settings within the repository settings page, including the root directory configuration.
Content Verification: I have cross-checked the content and structure of the HTML file and CSS against my local development environment, where the page works as expected.
Waiting Time: I have allowed sufficient time for GitHub Pages to update and deploy the changes after pushing updates to the repository.
Other Pages: The rest of my website is working as expected on GitHub Pages; the issue seems to be specific to the "Project & Skills" page.
Expected Outcome:
I expect the "Project & Skills" page to be accessible on my GitHub Pages site using the URL: https://abhijeet10sinha.github.io/AbhijeetSinha-Portfolio The page should display the content and structure as seen in my local development environment and when accessed using a local live server.

Actual Outcome:
Despite my efforts, when I access the "Project & Skills" page on GitHub Pages using the URL, it results in a "404 - Not Found" error. The page does not load as expected, and I'm unable to diagnose the issue based on the provided error message.

Next Steps Taken:
I have tried various troubleshooting steps, including checking file paths, verifying GitHub Pages settings, and waiting for deployment. However, none of these steps have resolved the issue. The page works perfectly when accessed from a local live server using my development environment.

[Redacted due to PII]

I would greatly appreciate your assistance in resolving this issue, as I have exhausted my available troubleshooting options. Thank you for your attention and support in resolving this matter.

Sincerely,
Abhijeet Kumar Sinha

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Hi @abhijeet10sinha,

I've removed your private information from this postas posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md). 

Additionally, you are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by mdtanvirahamedshanto** [comment]
responce #1

**Comment by habby1337** [comment]
hey, since you've already covered many bases, i'll offer a few additional suggestions that might help:

- **Check for typos and case sensitive** (just to double-check)
- **Relative vs Absolute paths** (Ensure that your file paths in the HTML and CSS files are using relative paths that are appropriate for your project structure and repository.)
- **File Extensions**: Make sure that your HTML file is indeed named with the .html extension and not something different like .htm.
- **404 Error Page**: Sometimes GitHub Pages might generate a 404 error page if there's an issue with your content. Review any custom 404 error page configurations to make sure they aren't interfering with the actual content.
- **Git Commit and Push**: Verify that you've committed and pushed all the necessary changes to your main (or appropriate) branch. (sometimes the issue can stem from not having the latest changes)
- **Check GitHub Pages Build Logs**: You can find them in the "Actions" tab of your repository.
- **Temporary Redirects**: If you've made changes to file paths or moved files around, GitHub Pages might still be serving cached content. 
- **Config File**: If you're using a Jekyll-powered site, check if you have a _config.yml file and whether it's configured correctly.
- **Disable GitHub Pages and Re-enable**: As a last resort, try disabling GitHub Pages for your repository, commit and push a minor change to your repository, and then re-enable GitHub Pages with the correct settings.

as a last resort:
- Check for open Issues or open one: If none of the above steps seem to work, consider opening an issue on GitHub's support page for GitHub Pages. Sometimes there might be specific issues with GitHub Pages deployments that the GitHub support team can assist with.
